ui/pages/confirmation/components/confirmation-network-switch/index.scss

```diff
@@ -0,0 +1,56 @@
+.confirmation-network-switch {
+  display: flex;
+  height: 100%;
+  justify-content: center;
+
+  &__result {
+    @include H3;
+
+    position: absolute;
+    top: 50%;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    text-align: center;
+    color: var(--Black-100);
+  }
+
+  &__icons {
+    display: flex;
+  }
+
+  &__icon {
+    display: block;
+
+    > h6 {
+      width: 64px;
+      position: absolute;
+      display: flex;
+      justify-content: center;
+    }
+
+    > img {
+      position: relative;
+    }
+
+    .icon-border {
+      background: var(--White-100);
+    }
+  }
+
+  &__center-icon {
+    @include H7;
```

Comment:
What text does this apply to?

Reply:
I was using this at first, but gave up on that solution. I forgot to remove it from scss, thank you for noticing. It is removed now.